README.md

```
@@ -1246,7 +1248,22 @@ int main() {
 }
 ```
 
-### Sorting Algorithms {#sorting-algorithms}
+### Matching
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)